packages/utils/package.json

```diff
@@ -36,7 +36,6 @@
     "abstract-level": "1.0.3",
     "bignumber.js": "9.0.2",
     "bn.js": "5.2.1",
-    "chalk": "5.2.0",
     "debug": "4.3.4",
     "ethers": "5.7.2",
     "futoin-hkdf": "1.5.0",
```

Comment:
Also this can be removed :) And `secp256k1` too

Reply:
Let's do a more general cleanup after the libp2p is merged though... this would get nasty really fast.